### PR TITLE
update argon2 dependency to argon2-cffi >= 19.2.0

### PR DESCRIPTION
**Pull request by fhoehle**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by fhoehle** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ requirements:
 
   run:
     - python
-    - argon2_cffi >=16.2
+    - argon2-cffi >=19.2.0
```
Comment (xhochy) [inline recipe/meta.yaml:29]: Is this a global name change?

Comment (fhoehle) [reply recipe/meta.yaml:29]: yes, `argon2<=19.1.X` is shipped by `argon2_cffi` and `argon2>=19.2.0` is shipped by `argon2-cffi`

**Review by xhochy — COMMENTED** [review]
@fhoehle can you add yourself as a maintainer and rerender?

**Comment by fhoehle** [comment]
@conda-forge-admin, please rerender

**Pull request merged**